README.md

```diff
@@ -7,6 +7,18 @@ This operator is in charge of handling the setup and configuration of the Giant
 It reconciles `cluster.cluster.x-k8s.io` objects and makes sure each `Cluster` is provided with:
 - TODO(atlas) update this section
 
+## Features
+
+### Grafan dashboards provisioning
+
+It will look for kubernetes `ConfigMaps` and use them as dashboards if they meet these criteria:
+- a label `app.giantswarm.io/kind: "dashboard"`
+- an annotation or label `giantswarm.io/organization` set to the organization the dasboard should be loaded in.
```

Comment:
I think we use this label for giant swarm organizations so we probably should not use this but smth like observability.giantswarm.io/organization instead. WDYT?